golos.charge/parameters.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <eosiolib/singleton.hpp>
+#include <common/parameter.hpp>
+#include <common/parameter_ops.hpp>
+
+namespace golos {
+
+    using namespace eosio;
+
+    struct vesting_acc : parameter {
+        name account;
+
+        void validate() const override {
+            eosio_assert(is_account(account), "Vesting account doesn't exist.");
+        }
+    };
+    using vest_acc_prm = param_wrapper<vesting_acc, 1>;
+
+
+    using charge_params = std::variant<vest_acc_prm>;
+
+    struct [[eosio::table]] charge_state {
+        vest_acc_prm vesting_acc_param;
```

Comment:
propose do not add `_param` suffix to field name, but instead put in to type name. the type clearly declares, that field is parameter, so there is no need to put that declaration to the name too.

actually, we have names inconsistency now: some contracts use this naming scheme, but others use `_prm` suffix for type and `_param` suffix for field name. this leads to confusion and errors when reading abi and use contracts…
This should be solved in separate issue, but in this issue please use the same naming style as surrounding parameters have (same with `st_` prefix as @s-medvedev mentioned earlier)